packages/core/shared.js

```diff
@@ -891,12 +895,12 @@ function checkUriSecurity(td) {
 
         }
 
-        // no security used non defined scheme, passed test
-        results.push({
-            "ID": "td-security-scheme-name",
-            "Status": "pass"
-        })
-        return results
+        // // no security used non defined scheme, passed test
```

Comment:
hey @egekorkan  sry for my late response.  MeanwhiIe  I changed project, but the linter told me, the code will not be reached, so thats why i commended it out. Which node version do you use? Also it would be great to have the Error Messages also in the Response like  "report, details, detailComments". But for now Im fine then! For changes or updated in the future please contact Felix again! Thank you